rosbag2_transport/test/rosbag2_transport/test_play_timing.cpp

```diff
@@ -190,3 +191,43 @@ TEST_F(PlayerTestFixture, play_ignores_invalid_delay)
   EXPECT_THAT(replay_time, Gt(lower_expected_duration));
   EXPECT_THAT(replay_time, Lt(upper_expected_duration));
 }
+
+TEST_F(PlayerTestFixture, play_respects_start_offset)
+{
+  rclcpp::Duration start_delay(0, static_cast<uint32_t>(5e8));  // 5e8 ns == 0.5 s
```

Comment:
```suggestion
  rclcpp::Duration start_delay(0, static_cast<uint32_t>(RCUTILS_S_TO_NS(0.5)));
```

Reply:
Addressed in https://github.com/ros2/rosbag2/pull/931/commits/07543e0c770db06fbada46268105fd4eea1a9d3a.